tgui/packages/tgui/interfaces/GasMixer/types.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * @file
+ * @copyright 2023
+ * @author Mordent (https://github.com/mordent-goonstation)
```

Comment:
Don't put my name on stuff I didn't make, lmao.

Reply:
Copy paste error which I've noticed and fixed like 1 minute *before* you reviewed!